### DISCUSSION TITLE
Undo button

### DISCUSSION THREAD

**Discussion by Department-stack** [TriliumNext/Trilium#6022] [Ideas] [answered]
### Describe feature

I wish that there was an undo button, for example while writing in a note, i may accidentally delete a paragrah. It would be nice if there was a shorcut or a button for me to be able to undo the last recent change in a note. This is a very fundamental feature that I need and without it i simply cannot use trilium unfortunately.

**Comment by eliandoran** [comment, marked as answer]
@Department-stack , for paragraph deletions, Ctrl+Z (undo) and Ctrl+Y (redo) should be a good start. If the change was made in the past, it's possible to restore content by going to the note revisions button.

![image](https://github.com/user-attachments/assets/ab160c38-141d-4149-9aad-8aa49fade5b9)

Are you having any issues with using them?

**Comment by Department-stack** [comment]
I did not realize this. Thank you for the help
